dags/asn_aggregates.py

```diff
@@ -0,0 +1,48 @@
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+from datetime import timedelta, datetime
+from utils.gcp import bigquery_etl_query
+
+default_args = {
+    "owner": "ascholtz@mozilla.com",
+    "email": ["ascholtz@mozilla.com", "tdsmith@mozilla.com",],
+    "depends_on_past": False,
+    "start_date": datetime(2020, 4, 5),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+dag = DAG("asn_aggregates", default_args=default_args, schedule_interval="@daily")
+
+asn_aggregates = bigquery_etl_query(
+    task_id="asn_aggregates",
+    destination_table="asn_aggregates_v1",
+    project_id="moz-fx-data-shared-prod",
+    dataset_id="telemetry_derived",
+    owner="ascholtz@mozilla.com",
+    email=["ascholtz@mozilla.com", "tdsmith@mozilla.com"],
+    parameters=("n_clients:INT64:500",),
+    dag=dag)
+
+wait_for_bq_events = ExternalTaskSensor(
+    task_id="wait_for_bq_events",
+    external_dag_id="main_summary",
```

Comment:
Both main_summary and copy_deduplicate DAGs use schedule `0 1 * * *` rather than `@daily` (which runs at midnight UTC). For ExternalTaskSensor to work, the schedules have to be exactly aligned. I'd recommend we use the same `0 1 * * *` schedule here. Otherwise, we need to provide an additional parameter to specify the offset between the schedules: `execution_delta=timedelta(hours=-1)`